webapp/portlet/src/main/webapp/dlp-quarantine/components/DlpQuarantineApp.vue

```diff
@@ -64,7 +64,8 @@
           <v-btn
             v-exo-tooltip.bottom.body="$t('documents.dlp.quarantine.previewDownload')"
             icon
-            text>
+            text
```

Comment:
This part should be rethinked. 
Here we can have a preview of a file, an activity, and a lot of thing, which are not documents.
We cannot make a dependency on documents here. 
We need to talk with Sameh